arch/sim/src/sim/up_devconsole.c

```diff
@@ -163,7 +289,20 @@ static int devconsole_poll(FAR struct file *filep, FAR struct pollfd *fds,
 
 void up_devconsole(void)
 {
-  register_driver("/dev/console", &devconsole_fops, 0666, NULL);
+  uart_register("/dev/console", &g_devconsole_dev);
+  uart_register("/dev/ttyS0", &g_devconsole_dev);
+}
+
+/****************************************************************************
+ * Name: up_devconsole
```

Comment:
wrong function name

Reply:
This name come from the original code wihout change, do you have a better name?